Abennat/DLS001.xml

```diff
@@ -51,7 +52,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="MA" when="2022-04-07">Created entity</change>
+            <change who="MA" when="2022-04-07">Created entity
+            <chnge who="Addisie" when="2022-04-15"></change>
```

Comment:
the tag is called "change" not chnge, and I believe you did not really change anything so I would not merge this pull request unless there are true changes

Did you check that your Atom works correctly? actually Atom should highlight errors and help you avoid them. Please follow the instructions in my last email.